HW 1/style.css

```diff
@@ -0,0 +1,54 @@
+#myBtn {
+    background-color: burlywood;
+    margin: 0;
+    padding: 0.5em 2em;
+    font-size: 30px;
+    align-items: center;
+    cursor: pointer;
+}
+
+.myStyle1{

```

Comment:
My style is not a proper name for a class. Please come up with more meaningful name. Then it would be easier to understand and support if it's a long running project.